requirements.txt

```diff
@@ -59,3 +59,4 @@ protobuf<=3.20.3, >=3.8.0
 contractions~=0.1.72
 fsspec~=2022.2.0
 google-api-core<=2.11.0
+openai<=0.27.7
```

Comment:
Now that we have the safe import, there is no need to install this for everyone. Please remove it.

Reply:
So without adding it to the the requirements.txt, the "build_website" job fails.

I asked about it earlier in the PR: "_So, it looks like the build_website still wants the package explicitly in requirements.txt. Should I put it in then or is it ok if that test doesn't pass?_" @klshuster said that it was ok to add in the followup. 

I don't have an opinion whether or not the package should be added or removed, but I just want to double check on whether or not its important that "build_website" needs to pass before the PR is merged.

Reply:
Sorry, I missed part of the previous conversation you had. It makes sense looking back at it.